Exception in Image running junit tests
java.lang.IllegalArgumentException: Argument not valid
at org.eclipse.swt.SWT.error(SWT.java:3845)
at org.eclipse.swt.SWT.error(SWT.java:3779)
at org.eclipse.swt.SWT.error(SWT.java:3750)
at org.eclipse.swt.graphics.ImageData.<init>(ImageData.java:440)
at org.eclipse.swt.graphics.ImageData.<init>(ImageData.java:284)
at org.eclipse.swt.graphics.Image.initNative(Image.java:1081)
at org.eclipse.swt.graphics.Image.<init>(Image.java:524)
at org.eclipse.swt.tests.junit.Test_org_eclipse_swt_graphics_Image.test_ConstructorLorg_eclipse_swt_graphics_DeviceLjava_lang_String(Test_org_eclipse_swt_graphics_Image.java:420)
at org.eclipse.swt.tests.junit.Test_org_eclipse_swt_graphics_Image.runTest(Test_org_eclipse_swt_graphics_Image.java:646)
at junit.framework.TestCase.runBare(TestCase.java:130)
at junit.framework.TestResult$1.protect(TestResult.java:106)
at junit.framework.TestResult.runProtected(TestResult.java:124)
at junit.framework.TestResult.run(TestResult.java:109)
at junit.framework.TestCase.run(TestCase.java:120)
at junit.framework.TestSuite.runTest(TestSuite.java:230)
at junit.framework.TestSuite.run(TestSuite.java:225)
at junit.framework.TestSuite.runTest(TestSuite.java:230)
at junit.framework.TestSuite.run(TestSuite.java:225)
at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:130)
at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)